tutorial/Lab2/README.md

```diff
@@ -0,0 +1,155 @@
+    $ kubectl create -f guestbook-service.yaml
+    ```
+    
+6. To check the updates, you can run ```kubectl get all --namespace default```:
+    
+    ```console
+    NAME                                READY     STATUS    RESTARTS   AGE
+    pod/guestbook-v1-7fc76dc46-9r4s7    1/1       Running   0          1h
+    pod/guestbook-v1-7fc76dc46-hspnk    1/1       Running   0          1h
+    pod/guestbook-v1-7fc76dc46-sxzkt    1/1       Running   0          1h
+    pod/redis-master-5d8b66464f-pvbl9   1/1       Running   0          1h
+    
+    NAME                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    service/guestbook      NodePort    172.21.45.29    <none>        3000:31989/TCP   31s
+    service/kubernetes     ClusterIP   172.21.0.1      <none>        443/TCP          9d
+    service/redis-master   ClusterIP   172.21.232.61   <none>        6379/TCP         1h
+    
+    NAME                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/guestbook-v1   3         3         3            3           1h
+    deployment.apps/redis-master   1         1         1            1           1h
+    
+    NAME                                      DESIRED   CURRENT   READY     AGE
+    replicaset.apps/guestbook-v1-7fc76dc46    3         3         3         1h
+    replicaset.apps/redis-master-5d8b66464f   1         1         1         1h
+    ```
+    Note: The service type has changed (to `NodePort`) and a new port has been allocated (`31989` in this output case) to the guestbook 
+    service. All `redis-slave` resources have been removed.
+
+5. View the guestbook as per [Lab1](../Lab1/README.md), using the updated port for the guestbook service.
+   
+# Update the application using Helm
+
+In this section, we'll update the previously deployed `guestbook-demo` application by using Helm.
+
+Before we start, let's take a few minutes to see how Helm simplifies the process compared to using Kubernetes. Helm's use of a [template language](https://docs.helm.sh/chart_template_guide/) provides great flexibility and power to chart authors, which removes the complexity to the chart user. In the guestbook example, we'll use the following capabilities of templating:
+* Values: An object that provides access to the values passed into the chart. An example of this is in `guestbook-service`, which contains the line `type: {{ .Values.service.type }}`. This line provides the capability to set the service type during an upgrade or install.
+* Control structures: Also called “actions” in template parlance, control structures provide the template author with the ability to control the flow of a template’s generation. An example of this is in `redis-slave-service`, which contains the line `{{- if .Values.redis.slaveEnabled -}}`. This line allows us to enable/disable the REDIS master/slave during an upgrade or install.
+
+Enough talking about the theory. Now let's give it a go!
+
+1. Update the application:
```

Comment:
I think you need to show, and talk to, the chart file with the conditional statement. The text in this section is fine, and when I was done reading it I understood what happened but w/o actually looking at the if-statement in the chart I think I would feel like I didn't get the complete concrete picture - just the theory of it.

Reply:
I think that is the idea, to give some insight to the reader so that they can go an investigate the files or the official Helm documentation. I would prefer this approach, rather than overload the lab with too much content. Is this ok?

Reply:
ok, added some content